### ISSUE TITLE
[Bug-gmx_MMPBSA_ana]: gmx_MMPBSA_ana does not open at the end of the calculation

### ISSUE THREAD

**Issue by hima111997** [Valdes-Tresanco-MS/gmx_MMPBSA#158]
### Bug summary

cant open gmx_mmpbsa_ana after finishing the analysis. However, it opens when i run it manually using gmx_MMPBSA_ana

### Terminal output

```bash
python: can't open file '/home/mario/PycharmProjects/gmx_MMPBSA/run_ana.py': [Errno 2] No such file or directory
```

### Actual outcome

python: can't open file '/home/mario/PycharmProjects/gmx_MMPBSA/run_ana.py': [Errno 2] No such file or directory

### Expected outcome

to open gmx_mmpbsa_ana

### Operating system

i am using ubuntu 20.04

### gmx_MMPBSA_ana Version

1.5.0

### Python version

3.7.10

### Installation

conda AmberTools + conda

**Comment by Valdes-Tresanco-MS** [comment]
Please, [update gmx_MMPBSA from development version](https://valdes-tresanco-ms.github.io/gmx_MMPBSA/installation/#installing-gmx_mmpbsa-v150)
`python -m pip install git+https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA -U`

**Comment by marioernestovaldes** [comment]
could you please send the files you are using, so we can reproduce this error?

thanks in advance

**Comment by hima111997** [comment]
the file that is used by gmx_MMPBSA_ana?

**Comment by marioernestovaldes** [comment]
the files that you are using in the command line... structure file, index, trajectory, etc

**Comment by hima111997** [comment]
[input_files.zip](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/8120361/input_files.zip)

the input file has the IE commented out but i calculated it for my system with ie_segment=25

**Comment by marioernestovaldes** [comment]
could you please end the *.itps associated with the topology file?

**Comment by hima111997** [comment]
[toppar.zip](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/8120424/toppar.zip)

**Comment by marioernestovaldes** [comment]
could you please double-check... seems like the toppar folder you sent is not the right one... the ligand doesn't even have the Cl atom

**Comment by hima111997** [comment]
sorry i uploaded the wrong file.
[toppar.zip](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/8120518/toppar.zip)

**Comment by Valdes-Tresanco-MS** [comment]
Definitively is a bug with the `interval` variable. We will fix it as soon as possible

**Comment by Valdes-Tresanco-MS** [comment]
Done... Please [update gmx_MMPBSA from the development version](https://valdes-tresanco-ms.github.io/gmx_MMPBSA/installation/#installing-gmx_mmpbsa-v150)
`python -m pip install git+https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA -U`

Fixed via [c4edb8f](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/commit/c4edb8ff5e508a21cfe6db77965763972813bae9)
